src/main/java/uk/gov/hmcts/reform/iacaseapi/domain/handlers/presubmit/AppealReferenceNumberHandler.java

```diff
@@ -69,6 +69,11 @@ public boolean canHandle(
 
         if (callback.getEvent() == Event.START_APPEAL) {
             asylumCase.write(APPEAL_REFERENCE_NUMBER, DRAFT);
+
+            String caseId = "" + callback.getCaseDetails().getId();
```

Comment:
Would `String caseId = String.valueOf(callback.getCaseDetails().getId())` not work in this case. 

You may not need to run `replaceAll` on it afterwards.

Reply:
Changed to String caseId = String.valueOf(callback.getCaseDetails().getId());